docs/tutorials/ollama-quickstart-docker.md

```diff
@@ -0,0 +1,35 @@
+# Ollama Quickstart with Docker
+
+1. Run the Ollama Docker container:
+
+```shell
+sudo docker run -d -v ollama:/root/.ollama -p 11434:11434 --name ollama ollama/ollama
+```
+
+For more detailed information, refer to the [Ollama Docker](https://hub.docker.com/r/ollama/ollama). Please note we are using CPU only, the AI will response slow, if you have GPU, you can follow the instruction to run the docker and using your GPU to improve performance.
+
+2. Pull the llama2 model:
+
+```shell
+curl --location 'http://localhost:11434/api/pull' \
+--header 'Content-Type: application/json' \
+--data '{
+    "name": "llama2:7b"
+}'
+```
```

Comment:
For the quickstart, it might be more straightforward to give instructions to exec into the container and run CLI commands instead of using the API from the host machine. I also think it might be easier to give a docker-compose example.

If so, this section would end up being something like
```sh
docker-compose exec ollama bash
```
```
root@c592ca62274b:/# ollama run llama2
```

Reply:
updated @dominickp